app/locustio/confluence/requests_params.py

```diff
@@ -0,0 +1,116 @@
+# flake8: noqa
```

Comment:
why do you need this here?

Reply:
There is a lot of `W605 invalid escape sequence` pep warning in this file in regular expressions.